templates/download/desktop/thank-you.html

```diff
@@ -40,15 +40,14 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Desktop</h1>
         </p>
         {% with version=version, system="desktop", architecture=architecture %}{% include "download/shared/_verify-checksums.html" %}{% endwith %}
         {% else %}
-        <h1>Thank you for your contribution</h1>
+        <h1>Thank you for your contributing</h1>
```

Comment:
Did you mean to write: "Thank you for contributing"?

Reply:
fixed...